uk.co.compucorp.civicrm.hrcore/backstop_data/page-objects/ssp/hr-resources.js

```diff
@@ -13,7 +13,7 @@ module.exports = (function () {
 
       casper.then(function () {
         casper.click('.fieldset-title');
-        casper.wait(2000);
+        casper.wait(500); // wait for CSS animation to complete
```

Comment:
you sure it's enough?

Reply:
No. I cannot be sure, same way as I cannot be sure in '5000', but it works for me. Also, this is not a CSS transition, I was wrong. I am now in doubts and I will leave 2000 like it was before.